src/Service/PaymentDetailsService.php

```diff
@@ -65,20 +65,19 @@ public function __construct(
     }
 
     /**
-     * @param array $details
+     * @param array $requestData
      * @param OrderTransactionEntity $orderTransaction
      * @return PaymentResponseHandlerResult
      * @throws PaymentFailedException
      */
     public function getPaymentDetails(
-        array $details,
+        array $requestData,
         OrderTransactionEntity $orderTransaction
     ): PaymentResponseHandlerResult {
-        $request = ['details' => $details];
 
         try {
             $this->checkoutService->startClient($orderTransaction->getOrder()->getSalesChannelId());
-            $response = $this->checkoutService->paymentsDetails($request);
+            $response = $this->checkoutService->paymentsDetails($requestData);
```

Comment:
`paymentData` is also required if it is returned from `/payments`
https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/payments/details__reqParam_paymentData